shared/modules/persistence_module.go

```diff
@@ -70,8 +70,7 @@ type PersistenceWriteContext interface {
 	SubtractPoolAmount(name string, amount string) error
 	SetPoolAmount(name string, amount string) error
 
-	// TODO(andrew): remove address from pool; pocket/issues/163
-	InsertPool(name string, address []byte, amount string) error
+	InsertPool(name string, address []byte, amount string) error // TODO (Andrew) remove address from pool #147
```

Comment:
Ditto about 147. We should scope down the issues

Reply:
A module by module cleanup issue may be scoped down, let that be tackled with #147 kindof counter intuitive to upscope this issue just to down scope others

Reply:
`let that be tackled with https://github.com/pokt-network/pocket/issues/147 kindof counter intuitive to upscope this issue just to down scope others`

Do you want me to resolve this as part of https://github.com/pokt-network/pocket/issues/147?

Reply:
Sorry - I understand the confusion now. I meant #149